docs/Concepts/NetworkID-And-ChainID.md

```diff
@@ -24,10 +24,16 @@ same and are specified in the genesis file.
 The network ID and chain ID are automatically defined by Besu when connecting to networks specified 
 using the [`--network`](../Reference/CLI/CLI-Syntax.md#network) option:
 
-- **MainNet:** chain-id 1, network-id 1
-- **Rinkeby:** chain-id 4, network-id 4
-- **Ropsten:** chain-id 3, network-id 3
-- **Dev:** chain-id 2018, network-id 2018
+| Network   | Chain ID | Network ID |
+|-----------|----------|------------|
+| `classic` | 61       | 1          |
+| `dev`     | 2018     | 2018       |
+| `goerli`  | 5        | 5          |
+| `kotti`   | 6        | 6          |
+| `mainnet` | 1        | 1          |
+| `mordor ` | 63       | 7          |
+| `rinkeby` | 4        | 4          |
+| `ropsten` | 3        | 3          |
 
```

Comment:
I don't think we should alphabetize them.  

Conceptually I feel we should keep the chain families together.  `mainnet`, `ropsten`, `rinkeby`, and `goerli` are all `ETH` chains, `classic`, `kotti`, and `mordor` are all `ETC` chains, then `dev` (an `ETH` chain).  Or maybe dev before the classic chains.  Perhaps a column for 'family' and all are "ETH" except the classic networks.

I think we also need a column for type, mainnet and classic are live or production networks (not sure of the term here).  ropsten, rinkeby, goerli, mordor, and kotti are testnets, dev is a development net.

mainnet
ropsten
rinkeby
goerli
dev
classic
mordor
kotti

Reply:
I added a Chain column to group together ETH networks and ETC networks and a Type column to label production, test, and development networks.